test.md

```diff
@@ -76,5 +76,22 @@ This simply checks that the given function exists in the `<funcs>` cell and has
            )
            ...
          </funcs>
+```
+
+### Memory Assertions
+
+Checks that memory is in a certain state
+
+```k
+    syntax Instr ::= "#assertMemory" Int Int Bytes String
+ // -----------------------------------------------------------------------
+    rule <k> #assertMemory MIN MAX DATA DESCRIPTION => . ... </k>
+         <mems>
+           <memInst>
+              <min> MIN </min>
+              <max> MAX </max>
+              <content> DATA </content>
+           </memInst> => .Bag
```

Comment:
I would prefer formatting more like this:

```
         ( <memInst>
              <min> MIN </min>
              <max> MAX </max>
              <content> DATA </content>
           </memInst>
        => .Bag
         )
```

Also, do you want a `...` inside the `<mems>` to capture any additional `<memInst>` that may exist?

Reply:
The current version of WASM seems to only support one memory instance. So either there is no memory at all, or there is a memory instance. I reflected this in the configuration as:
`<memInst multiplicity="?">   //current WASM supports at most 1 memory instance`.

Reply:
Hmmmm, in that case it makes sense. Then we can just switch `multiplicity="*"` to get multiple memory instance support. Please make the formatting change as well.